surveys/sphinx-survey-2025.yml

```diff
@@ -110,27 +122,41 @@ questions:
     multiple: true
     other: true
     choices:
+      - "Cloudflare Pages"
+      - "DigitalOcean"
       - "GitHub Pages"
+      - "Hetzner" 
       - "Netlify"
       - "Read The Docs"
-      - "My institution has its own deployment"
+      - "Render"
+      - "Vercel" 
+      - "Vultr"
+      - "We have our own deployment"
+      - "As a part of an existing website"
+      - "I don't know"
 
   - title: "What markup language are you using in your documentation?"
     multiple: true
     other: true
     choices:
-      - "Markdown"
+      - "AsciiDoc"
+      - "Confluence/Jira"
+      - "Markdown - CommonMark"
+      - "Markdown - GitHub Flavored Markdown"
```

Comment:
We are repeating markdown here. 
For consistency this should be changed to Markdown - GitHub